WIP - Contact sales page

chanhman opened the pull request:
This PR adds a new page: Contact Sales.

chanhman commented:
What's left:
- Implement floating labels
- Style Checkbox
- Add custom select box, eg. don't use <select>

gib commented:
Looking good @chanhman! I'll pull this down later and take a look before I add any comments.

chanhman commented:
What's left:
- Add custom selectbox
- Will just add Contact Page while I'm at it because it will share some styles with this branch

sandersonet commented on source/javascripts/floating-labels.coffee:
```diff
@@ -0,0 +1,8 @@
+$('.floating-label .input').keyup ->
```
Hey @chanhman,

Couple quick notes about this JS.

Repeatedly calling `$(this)`, `.closest()`, and `.find()`  is costly, as jQuery has to query the DOM for each of those calls.  It becomes especially worse when doing these calls from the `keyup` callback.  

A quick fix here would be to cache the elements you need prior to responding to `keyup`.

I've re-written the codepen you used into a jQuery plugin that does just that here:  https://codepen.io/anon/pen/jGrOJM

```javascript
$(document).ready(function() {
  $.fn.extend({
    floatingLabel: function(_options) {
      $(this).each(function() {
        $this = $(this);
        $input = $this.find("input");
        $label = $this.find("label, .placeholder");

        $input.keyup(function(e) {
          var hasValue = e.target.value.length;
          if (hasValue) {
            $label.addClass("is-visible active");
          } else {
            $label.removeClass("is-visible active");
          }
        });
      });
    }
  });

  $(".empty-val-input").floatingLabel();
});
```


You'll need to update the classnames to match yours (and feel free to convert to coffeescript)

chanhman commented on source/contact/index.haml:
```diff
@@ -0,0 +1,26 @@
+    .contact-cards
+      .grid-container.grid--2up
+        %a.grid-item.card{:href => "/contact/sales"}
+          .card__title.card__title--centered
```
@henryhund I used the example site's content for placement holders. Let me know if you have any existing content you would like me to use.

chanhman commented:
@gib I updated the drop-down feature I pushed yesterday. I implemented a hidden input field that excepts a value from the drop-down menu. Unfortunately, I couldn't figure out how to make it function like a select box and I don't want to spend(waste) more time on it; especially if you could do it faster. Maybe we can go over it today?

chanhman commented:
@gib This PR is now up to date with the edits we talked about. The only thing left is hooking up the form.
